cdap-ui/app/cdap/components/FieldLevelLineage/OperationsModal/ModalContent.js

```diff
@@ -26,6 +26,7 @@ import T from 'i18n-react';
 const PREFIX = 'features.FieldLevelLineage.OperationsModal';
 
 function formatDatasets(datasets) {
+  // return in form 'dataset1, dataset2, and dataset3'
```

Comment:
question, this comment seemed different than what the code is doing. Which one should be the correct one?

Reply:
Ahh, this comment was meant to go with the v2 modal (which is not this one). I ended up only made the change in syntax to v2, will update the comments accordingly.